### ISSUE TITLE
if speed isn't captured for interfaces, we can get false asserts because of speed mismatch 

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#294]
**Describe the bug**
this means cumulus data

spun up cloud native datacenter, dual topology, bgp

```
jpietsch> interface unique columns=speed                                                                                                                             Error: -1        speed  count
0           0     59
1        1000     63
2        2000      8
3  4294967295      8
```
this is especially bad because any of these interfaces with 0 connected to something non-zero has an interface assert failure.

**Comment by ddutt** [comment]
This is because there is no easy way to capture speed of an interface on a Linux server.

**Comment by jopietsch** [comment]
ok, that makes sense, though it's disappointing. 
what should we do about the false asserts because of this?
from basicdualbgp
```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli interface assert --status=fail
  namespace hostname ifname state peerHostname peerIfname assert      assertReason
0  dual-bgp   edge01   eth1    up       exit01       swp5   fail  [Speed mismatch]
1  dual-bgp   edge01   eth2    up       exit02       swp5   fail  [Speed mismatch]
2  dual-bgp   exit01   swp5    up       edge01       eth1   fail  [Speed mismatch]
3  dual-bgp   exit02   swp5    up       edge01       eth2   fail  [Speed mismatch]
Assert failed
```

**Comment by LucaNicosia** [comment]
I will be working on this bug

**Comment by ddutt** [comment]
Fixed in 0.16.0 alpha
